webapi/src/test/scala/org/knora/webapi/e2e/v2/SearchRouteV2R2RSpec.scala

```diff
@@ -1869,7 +1912,7 @@ class SearchRouteV2R2RSpec extends R2RSpec {
 
         }
 
-        "search for a anything:Thing with a list value" in {
+        "search for an anything:Thing with a list value" ignore { // ignored because the list node label returned is not deterministic
```

Comment:
I guess we should get rid of the list node label and query the list v2 route instead.

Reply:
Yes, you wanted to do that in another PR, right?

Reply:
Yes, #763 would be a good occasion to that. Could we work on it together?

Reply:
Yes please!